octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -751,6 +757,15 @@ def list_findings(
         elif only_muted:
             muted_clause = "[?muted_finding :MutedFinding/finding ?finding]"
 
+        search_statement = (
+            f"""[?finding :xt/id ?id]
+                                [(clojure.string/includes? ?id \"{escape_string(search_string)}\")]"""
```

Comment:
```suggestion
                                [(str/lower-case (clojure.string/includes? ?id \"{escape_string(search_string)}\"))]"""
```

And also lowercase the input search_string

Reply:
Have you tested that? It doesnt work for me.

Reply:
Nope, sorry